src/remote-desktop.c

```diff
@@ -59,11 +59,6 @@ G_DEFINE_TYPE_WITH_CODE (RemoteDesktop, remote_desktop,
 typedef enum _RemoteDesktopSessionState
```

Comment:
Commit message should be e.g. `remote-desktop: Simplify state machine`

with e.g.

```
Remove some state that didn't help much keeping track of separately. We'll
want to be able to prepare more in an arbitrary order before starting so
just have a single state for not having started (`INIT`).
```